templates/default/sv-jenkins-run.erb

```diff
@@ -6,10 +6,11 @@
 #
 
 
+<% groups = node['jenkins']['master']['groups'].join(':') %>
 exec 2>&1
 cd <%= node['jenkins']['master']['home'] %>
 touch jenkins.start
-exec chpst -u <%= node['jenkins']['master']['user'] %> -U <%= node['jenkins']['master']['user'] %> \
+exec chpst -u <%= groups %> -U <%= groups %> \
```

Comment:
Not in all situations. We need to ensure user and group are respected as separate tuneables. I think you want something like:

```ruby
exec chpst -u <%= node['jenkins']['master']['user'] %>:<%= node['jenkins']['master']['runit']['groups'].join(':') %> -U <%= node['jenkins']['master']['user'] %>:<%= node['jenkins']['master']['runit']['groups'].join(':') %>
```